lib/src/error.ts

```diff
@@ -1,3 +1,6 @@
+import { ExchangeType } from "@ledgerhq/wallet-api-client";
+import { ErrorOrigin } from "./sdk";
```

Comment:
This dependency create a cycle one between `error.ts` and `sdk.ts`.
We can either replace the parameter as a string or if you want to enforce the use of an enum, extract this enum definition in a different file.